.github/workflows/wheel-linux-x86-64.yaml

```diff
@@ -252,8 +254,7 @@ jobs:
         cp --dereference $GITHUB_WORKSPACE/llvm-build/lib/libmlir_float16_utils.so.* $GITHUB_WORKSPACE/frontend/catalyst/lib
         cp --dereference $GITHUB_WORKSPACE/llvm-build/lib/libmlir_c_runner_utils.so* $GITHUB_WORKSPACE/frontend/catalyst/lib
         # Copy mlir bindings to frontend/mlir_quantum
-        mkdir -p $GITHUB_WORKSPACE/frontend/mlir_quantum
-        cp -R --dereference $GITHUB_WORKSPACE/quantum-build/python_packages/quantum/mlir_quantum/runtime $GITHUB_WORKSPACE/frontend/mlir_quantum/runtime
+        cp -R --dereference $GITHUB_WORKSPACE/quantum-build/python_packages/quantum/mlir_quantum/ $GITHUB_WORKSPACE/frontend/mlir_quantum/
```

Comment:
I think we should do a more fine-grained include here so as not to unnecessarily blow up the wheel size. The shared libs in the bindings are actually quite large (74MB uncompressed), whereas everything else is a few kilobytes..

Reply:
See [here](https://github.com/PennyLaneAI/catalyst/pull/40/commits/97100621afe49528fb5182ff32bfbc56de20dd42) and [here](https://github.com/PennyLaneAI/catalyst/pull/40/commits/e111b8719bcf39e55d63778abd6d9002f5d31ef5) and the same logic being applied in the following commits.

Reply:
Very nice 👍